.circleci/extended/orb-test-job.yml

```diff
@@ -0,0 +1,12 @@
+# This config is equivalent to both the '.circleci/extended/orb-free.yml' and the base '.circleci/config.yml'
+version: 2.1
+
+orbs:
+  node: circleci/node@4.1
+
+workflows:
+  sample:
+    jobs:
+      - node/test:
+          # this is the node version to use for the `cimg/node` tag
+          version: "15.1"
```

Comment:
Could be useful to link to cimg documentation: https://circleci.com/developer/images/image/cimg/node